examples/feature_selection/plot_permutation_test_for_classification.py

```diff
@@ -22,48 +20,115 @@
 from sklearn.svm import SVC
 from sklearn.model_selection import StratifiedKFold
 from sklearn.model_selection import permutation_test_score
-from sklearn import datasets
```

Comment:
Could you move all imports next to the cell where they will be used. It is a bit more intuitive when reading the notebook.